lab_1_keywords_tfidf/start.py

```diff
@@ -5,20 +5,57 @@
 # pylint:disable=too-many-locals, unused-argument, unused-variable, invalid-name, duplicate-code
 from json import load
 
+from lab_1_keywords_tfidf.main import (
+    calculate_chi_values,
+    calculate_expected_frequency,
+    calculate_frequencies,
+    calculate_tf,
+    calculate_tfidf,
+    clean_and_tokenize,
+    extract_significant_words,
+    get_top_n,
+    remove_stop_words,
+)
+
 
 def main() -> None:
     """
     Launches an implementation.
     """
+    cleaned_tokens = None
+    frequencies = None
+    tf_values = None
+    tfidf_values = None
+    alpha = 0.001
+    expected_frequency = None
+    chi_values = None
+    result = None
     with open("assets/Дюймовочка.txt", "r", encoding="utf-8") as file:
         target_text = file.read()
+    tokens = clean_and_tokenize(target_text)
     with open("assets/stop_words.txt", "r", encoding="utf-8") as file:
         stop_words = file.read().split("\n")
+    if tokens is not None:
```

Comment:
It is better to use `if not tokens: return`, then you will not need to create variables and set them as `None`